docs/backend.md

```diff
@@ -0,0 +1,138 @@
+# Backend design doc
+
+This is a design doc proposing some possible approaches for an ATIP backend. It doesn't represent any final decisions or describe anything actually built.
+
+As of June 2023, ATIP is a web app for editing GeoJSON data. Those GeoJSON files are stored in each user's browser local storage and manually saved/loaded as local files. There is no central place to submit and store files, or any concept around tracking changes to a file.
+
+Terminology:
+
+- a **scheme** is a single ATIP GeoJSON file, with a list of **interventions** or **features** (these are used interchangeably)
+- **ATE** is the central group evaluating and inspecting schemes
+- **LAs** (local authorities) are the groups submitting schemes for some area. (The exact geographic scope doesn't matter; these could be transport or highway authorities or something else)
```

Comment:
```suggestion
- A **scheme** is one or more infrastructure interventions represented as a single ATIP GeoJSON file.
- An **intervention** is a single point (e.g. crossing), line (e.g. route) or area (e.g. area-wide traffic management intervention) that is part of a scheme.
- **Authorities** are the groups submitting schemes for some area. 
```

Reply:
I just used the `suggestion` syntax for the first time 🎉 

In terms of the suggestion: I think we should keep docs in here as non England specific as possible. Short, generalisable, simple and easy to maintain is my thinking. We can add ATE specific content for the version that goes on our website but that's not happening for a while. Local authorities is a term that has multiple meanings (including local authority districts and our transport authorities) so we should avoid, I've suggested just authorities here. And 👎  to using acronyms, let's keep it as readable as possible.